dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTrackerHelper.h

```diff
@@ -0,0 +1,60 @@
+﻿#pragma once
+#include <windows.h>
+#include <wil/result.h>
+#include <appmodel.h>
+#include <Microsoft.Windows.System.EnvironmentManager.h>
+
+using namespace winrt::Microsoft::Windows::System::implementation;
+
+static bool DoesChangeTrackingKeyExist()
+{
+    std::filesystem::path subKey = std::filesystem::path{ LR"(Software\Microsoft\AppModel\RegistryWriteVirtualization\ExcludedKeys\HKEY_CURRENT_USER/Software/ChangeTracker)" };
+    wil::unique_hkey keyToChangeTracker{};
+    LSTATUS openResult{ RegOpenKeyEx(HKEY_LOCAL_MACHINE, subKey.c_str(), 0, KEY_READ, keyToChangeTracker.put()) };
+    if (openResult == ERROR_SUCCESS)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+static std::wstring PackageFullName()
+{
+    WCHAR packageFullName[PACKAGE_FULL_NAME_MAX_LENGTH + 1]{};
```

Comment:
Even better, add to `dev\ccommon\appmodel.identity.h` (below)

```
inline std::wstring GetCurrentPackageFullName()
{
    WCHAR packageFullName[PACKAGE_FULL_NAME_MAX_LENGTH + 1]{};
    UINT32 n{ ARRAYSIZE(packageFullName) };
    const auto rc{ ::GetCurrentPackageFullName(&n, pacckageFullName) };
    if (rc == ERROR_SUCCESS)
    {
        return std::wstring{ packageFullName };
    }
    else if (rc == APPMODEL_ERROR_NO_PACKAGE)
    {
        return std::wstring();
    }
    else
    {
        THROW_HR_MSG(HRESULT_FROM_WIN32(rc), "GetCurrentPackageFullName rc=%d", rc);
    }
}
```

then delete and change callers like 46 from

```
auto packageFullName{ PackageFullName() };
```

to

```
auto packageFullName{ ::AppModel::Identity::GetPackageFullName() };
```